common/rabbitmq/templates/bin/_rabbitmq-start.tpl

```diff
@@ -53,22 +32,91 @@ rabbitmqctl trace_on
 rabbitmqctl enable_feature_flag all
 {{- end }}
 
-eval $(timeout 5.0 rabbitmqctl list_users -q | awk '{printf "users[\"%s\"]=\"%s\"\n", $1, substr($2, 2, length($2)-2)}')
+python3 - <<EOF
```

Comment:
Please move the Python code into a dedicated file to make easier to maintain...

Reply:
Moved script to separate file